.eslintrc

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
AVA only was installed for a second. I was going to replace Tape with AVA but it wasn't worth it.